src/index.js

```diff
@@ -134,6 +142,15 @@ CerebroConfig.prototype.getRawConfig = function() {
     return this._resolved;
 };
 
+/**
+ * Returns the labels from the entries
```

Comment:
Can you add more detail to this API doc? What is the expected return object shape? What is this meant to be used for?